### PR TITLE
meson.build: when in release tarball, default to 0

### PR DESCRIPTION
**Pull request by ret2libc**

### CODE REVIEW

File: libr/bin/meson.build [hunk 1]
```diff
@@ -131,6 +131,7 @@ r_bin = library('r_bin', files,
     r_magic,
     libr_shlr
   ],
+  dependencies: lmagics,
```
Comment (xarkes) [inline libr/bin/meson.build]: Why no `[]`?

File: libr/core/meson.build [hunk 2]
```diff
@@ -97,7 +97,7 @@ r_core = library('r_core', files,
     r_bp,
     libr_shlr
   ],
-  dependencies: [platform_deps],
+  dependencies: [platform_deps] + lmagics,
```
Comment (xarkes) [inline libr/core/meson.build]: Why a `+`, can't you put it in the array?

**Comment by xarkes** [comment]
I think @pelijah can give his opinion too.

File: libr/bin/meson.build (hunk 1 above)
Comment (ret2libc) [reply libr/bin/meson.build]: I'm no expert of `meson`, but `lmagics` is already an array, so I didn't put `[]`.

File: libr/core/meson.build (hunk 2 above)
Comment (ret2libc) [reply libr/core/meson.build]: Again, `lmagics` is already an array, so I just appended it to `[platform_deps]`. Does it work even if i put it inside the array?

**Comment by ret2libc** [comment]
BTW, I will probably reimplement this in such a way that even when capstone is not used from system, it can be just expressed as a dependency (see http://mesonbuild.com/Dependencies.html#declaring-your-own)

**Comment by ret2libc** [comment]
Superseded by https://github.com/radare/radare2/pull/10617

**Pull request closed without merging**